balloc/balloc.c

```diff
@@ -69,40 +70,154 @@ struct balloc_allocation_context {
 	struct m0_ext                  bac_final;/*< final results */
 };
 
-static inline int btree_lookup_sync(struct m0_be_btree  *tree,
-			       const struct m0_buf *key,
-			       struct m0_buf       *val)
+static int btree_lookup_callback(struct m0_btree_cb  *cb,
+				 struct m0_btree_rec *rec)
 {
-	return M0_BE_OP_SYNC_RET(op, m0_be_btree_lookup(tree, &op, key, val),
-	                         bo_u.u_btree.t_rc);
+        struct m0_btree_rec     *datum = cb->c_datum;
```

Comment:
Use of spaces instead of tabs in this function.
Please check once.

Reply:
Done.